invenio_orcid/tasks.py

```diff
@@ -128,16 +141,20 @@ def send_to_orcid(sender, api=None):
                     else:
                         api.update_record(author_orcid, token,
                                           'work', orcid_json, str(put_code))
-                    logger.info("Succersfully sent " +
-                                sender.get('control_number') + " to orcid.")
+                    current_app.logger.info(
+                        "Succersfully sent {0} to orcid.").format(
```

Comment:
Pep8 complains for long lines if I use the same style as line 117.
Should I change 117 to follow the others?